packages/api/lambdas/sf-scheduler.js

```diff
@@ -33,9 +42,9 @@ const getCollection = (collection) => {
  * @returns {Promise}
  */
 async function handleScheduleEvent(event) {
-  const [providerRecord, collection] = await Promise.all([
+  const [provider, collection] = await Promise.all([
```

Comment:
I think this is a breaking change? Line 50 below is expecting a variable named providerRecord named `provider`

Reply:
I believe this comment is outdated.